web-server/v0.4/src/pages/Explore/index.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Table from '@/components/Table';
+import Explore from './index';
+
+const mockProps = {};
+
+const mockDispatch = jest.fn();
+configure({ adapter: new Adapter() });
+const wrapper = shallow(<Explore.WrappedComponent dispatch={mockDispatch} {...mockProps} />, {
+  disableLifecycleMethods: true,
+});
+
+describe('test Summary page component', () => {
```

Comment:
We should specify the `Explore` page component here in the test description.

Reply:
Corrected!